config/application.yml.default

```diff
@@ -127,7 +127,7 @@ idv_send_link_max_attempts: 5
 ie11_support_end_date: '2022-12-31'
 idv_sp_required: false
 in_person_cta_variant_testing_enabled: true
-in_person_cta_variant_testing_percents: '{"A":50, "B":50}'
+in_person_cta_variant_testing_percents: '{"A":12.5, "B":12.5, "C":75}'
```

Comment:
Will this cause our automated feature tests to fail 75% of the time, since they are looking for the in-person proofing CTA?

I also wonder if we really want this to be the default. Anyone doing local testing will need to work around 3 different buckets. I _think_ that having the default be more like `A: 100, B, 0, C: 0` would make more sense and then just updating staging and prod to have the configuration set here

Reply:
Reason being that people are generally trying to test a specific thing when developing locally. At best they would need to override this default value in `config/application.yml`. At worst they wouldn't be aware of our A/B/C testing and it would lead to people (potentially on other teams) getting really confused about what's happening in the app

Reply:
> Will this cause our automated feature tests to fail 75% of the time

I don't believe so. The IPP CTA in variants A and B is also not found by the original test; I'll double check but I believe the test updates included in LG-8274 covered this.

> Anyone doing local testing will need to work around 3 different buckets

Developers can set this config value in their local application.yml to 100% whichever bucket they need to be working in. Is our general practice to set development values in the default prod config and directly augment staging and prod after merge? That doesn't sound like a good process to me but I'm open to other perspectives.

Reply:
>Is our general practice to set development values in the default prod config and directly augment staging and prod after merge?

What I suggest is:
1. Set the root default to be 100% variant a which would be used for local dev and the automated test suite
2. Set the production category default (which includes all deployed environments) to 12.5%, 12.5%, 75%
3. Only augment in deployed environments if/when we need to